### PR TITLE
Add GateFilter to RadarDisplay

### PR DESCRIPTION
**Pull request by nguy**
Add GateFilter instance (pyart.correct.GateFilter) as a keyword input to ```plot_ppi``` and ```plot_rhi``` in  ```radardisplay.py```.
If acceptable, it could be extended to the other display code. It's a relatively simple change. It will make building the ARTView masking procedure much easier and allow users to see changes of applying a masking filter to data.

### CODE REVIEW

File: pyart/graph/radardisplay.py [hunk 1]
```diff
@@ -320,6 +321,9 @@ def plot_ppi(self, field, sweep=0, mask_tuple=None, vmin=None, vmax=None,
             coordinates themselved as the gate edges, resulting in a plot
             in which the last gate in each ray and the entire last ray are not
             plotted.
+        gatefilter : GateFilter Instance
+            pyart.correct.filters.GateFilter instance. None will result in no
```
Comment (jjhelmus) [inline pyart/graph/radardisplay.py]: Just use "GateFilter instance" here, the namespace is not important

File: pyart/graph/radardisplay.py [hunk 2]
```diff
@@ -320,6 +321,9 @@ def plot_ppi(self, field, sweep=0, mask_tuple=None, vmin=None, vmax=None,
             coordinates themselved as the gate edges, resulting in a plot
             in which the last gate in each ray and the entire last ray are not
             plotted.
+        gatefilter : GateFilter Instance
```
Comment (jjhelmus) [inline pyart/graph/radardisplay.py]: Remove Instance

File: pyart/graph/radardisplay.py [hunk 3]
```diff
@@ -421,6 +427,9 @@ def plot_rhi(self, field, sweep=0, mask_tuple=None, vmin=None, vmax=None,
             coordinates themselved as the gate edges, resulting in a plot
             in which the last gate in each ray and the entire last ray are not
             not plotted.
+        gatefilter : GateFilter Instance
+            pyart.correct.filters.GateFilter instance. None will result in no
```
Comment (jjhelmus) [inline pyart/graph/radardisplay.py]: Some as above

File: pyart/graph/radardisplay_airborne.py [hunk 4]
```diff
@@ -221,6 +221,9 @@ def plot_sweep_grid(self, field, sweep=0, mask_tuple=None, vmin=None,
             coordinates themselved as the gate edges, resulting in a plot
             in which the last gate in each ray and the entire last ray are not
             plotted.
+        gatefilter : GateFilter Instance
+            pyart.correct.filters.GateFilter instance. None will result in no
+            gatefilter mask being applied to data.
```
Comment (jjhelmus) [inline pyart/graph/radardisplay_airborne.py]: Same as above

**Comment by jjhelmus** [comment]
Adjust the docstring and I think this should be good to go.

**Comment by nguy** [comment]
Changes also made in the ```radarmapdisplay.py``` and ```radardisplay_airborne```

**Comment by jjhelmus** [comment]
Looks good, will merge once AppVeyor gives the green light.  Thanks for all the work @nguy!

**Pull request merged**